src/components/Recipe/Recipe.js

```diff
@@ -1,14 +1,20 @@
 import styled from 'styled-components/macro'
 import PropTypes from 'prop-types'
-export default function Recipe({
-  image,
-  title,
-  calories,
-  ingredients,
-  servings,
-}) {
+import { NavLink } from 'react-router-dom'
+export default function Recipe({ recipe }) {
+  const { image, calories } = recipe
+  const title = recipe.label
+  const servings = recipe.yield
+  const ingredients = recipe.ingredientLines.length
+  const recipeId = recipe.id
```

Comment:
You could destructure title, servings, ingredients and id as well by writing `const {image, calories, label : title, yield : servings, ...`